tests/integration_test/run_algorithm.py

```diff
@@ -0,0 +1,123 @@
+# Util imports
+from input_output.file_output import load_graph_list_from_filepath
+from input_output.sys_output import passed, fail
+import sys
+import tkinter as tk
+from tkinter import filedialog
+from time import time
+from tests.integration_test.tournament_output import tournament_output
+from tests.integration_test.test_output import test_output
+# Algorithm imports
+from tests.integration_test.settings import *
+from tests.integration_test.isomorphism_problem import are_isomorph, amount_of_isomorphisms
+
+"""
+General integration test for the Graph Isomorphisms problem.
+Settings of the algorithm must be changed in > settings.py <
+"""
+
+"""
+FILE INPUT
+"""
+# Select files to evaluate
+root = tk.Tk()
+root.withdraw()
+file_paths = filedialog.askopenfilenames()
+
+"""
+RUN ALGORITHM
+"""
+if run_mode == 1:
+    error_count = 0
+for file_path in file_paths:
+    graphs = load_graph_list_from_filepath(file_path)
+    filename = (file_path.split("/")[-1]).split(".")[0]
+
+    # Some data structures that are used to determine if graphs are isomorphic more efficiently
+    isomorphisms = []       # List of lists that saves all isomorphic pairs (or more than 2, if that is the case)
+    iso_count = {}          # Dictionary that saves for each graph the amount of automorphisms
+    total_time = 0          # Total processing time of the graphs of this file
+    skip = [False for _ in range(len(graphs))]  # To check if you can skip a cycle
+
+    # In this first loop, for each combination, it is determined if they are isomorphic or not
+    for i in range(len(graphs) - 1):
+        for j in range(i + 1, len(graphs)):
+            s = filename + ": Determining if [" + str(i) + "," + str(j) + "] are isomorphic (out of " + str(len(graphs) - 1) + " graphs)"
```

Comment:
Ik zie deze regels helemaal niet op de console verschijnen.

Reply:
ik zie deze regels wel bij modulesC, omdat er wat meer tijd tussen de berekeningen zit.
Cool gevonden !

Reply:
Met welke file heb je getest? Want dit is een regel die steeds overschreven wordt. Dus bij torus24 gaat het zo snel dat je dat mist. Maar bij products72 bijvoorbeeld kun je zien waar die op dat moment dan mee bezig is

Reply:
Oh, heb alleen met de snelle gedaan, maar goede toevoeging!

Reply:
ziet er heel cool uit in tournament mode 👍